filtered-resource-service/src/component-tests/java/uk/gov/gchq/palisade/component/filteredresource/repository/MapTokenErrorMessagePersistenceLayer.java

```diff
@@ -17,7 +17,7 @@
 package uk.gov.gchq.palisade.component.filteredresource.repository;
 
 import uk.gov.gchq.palisade.service.filteredresource.domain.TokenErrorMessageEntity;
-import uk.gov.gchq.palisade.service.filteredresource.repository.exception.TokenErrorMessagePersistenceLayer;
+import uk.gov.gchq.palisade.service.filteredresource.repository.error.TokenErrorMessagePersistenceLayer;
```

Comment:
Worth a double-check.  We have exception as our default package for all our bespoke Exceptions.  Here it looks like  a change to that convention.

Reply:
Yes, this is for the actor system that monitors the error topic in order to communicate with the client, similar to the offset package and the masked-resource-offset topic